app/Api/V1/Domain/Problem/Param/CreateProblemParam.php

```diff
@@ -17,7 +17,7 @@ class CreateProblemParam
     const QUERY_PARAM_VALIDATION = [
         Problem::ATTRIBUTE_SLUG => [
             'required',
-            'regex:/^[a-z0-9]+(?:-[a-z0-9]+)*$/',
+            'regex:/^[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*$/',
```

Comment:
I think we need to use UPPERCASE only or lowercase only
I prefer UPPERCASE only, you can remove the lowercase character

Reply:
Roger that 🙆